### ISSUE TITLE
bug: The latest image seems to have UI issue

### ISSUE THREAD

**Issue by hypernovas** [langfuse/langfuse#1549]
### Describe the bug

I upgrade from v2.14.6-0 to the latest (we try to test OKTA integration with this [PR ](https://github.com/langfuse/langfuse/pull/506/files)included), the frontend does not work:
Screenshot:
<img width="246" alt="Screenshot 2024-03-29 at 4 27 01 PM" src="https://github.com/langfuse/langfuse/assets/17284665/7dfb9b84-ca63-44da-b87d-d9654a1cda24">

The server log looks fine:
<img width="924" alt="Screenshot 2024-03-29 at 4 29 40 PM" src="https://github.com/langfuse/langfuse/assets/17284665/e3074e01-3383-4366-906b-b4ee830996b8">

### To reproduce

The kubernetes deployment yaml is the following:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: langfuse-deployment
  labels:
    app: langfuse
spec:
  replicas: 1
  selector:
    matchLabels:
      app: langfuse
  template:
    metadata:
      labels:
        app: langfuse
    spec:
      containers:
      - name: langfuse-container
        image: ghcr.io/langfuse/langfuse:latest
        ports:
        - containerPort: 3000
        env:
        - name: DATABASE_URL
          value: "postgresql://postgres:password@{our-aws-RDS-db}.us-west-2.rds.amazonaws.com:5432/langfuse"
        - name: NEXTAUTH_URL
          value: "http://{host_URL}:3000"
        - name: NEXTAUTH_SECRET
          value: "mysecret"
        - name: SALT
          value: "mysalt"
---
apiVersion: v1
kind: Service
metadata:
  name: langfuse-svc
spec:
  selector:
    app: langfuse
  ports:
  - protocol: TCP
    port: 3000
    targetPort: 3000
```

**Comment by marcklingen** [comment]
Can you share whether there are any errors in the browser console?

Are you on the most recent version?

This seems to be related to #1539

**Comment by marcklingen** [comment]
Just in case this is a production system, I’d recommend to pin the major version (2)

**Comment by marcklingen** [comment]
This could be related to CSP headers, added in  https://github.com/langfuse/langfuse/pull/1532

If yes, there should be errors in the browser console

**Comment by hypernovas** [comment]
There are errors:
<img width="746" alt="Screenshot 2024-03-29 at 6 18 16 PM" src="https://github.com/langfuse/langfuse/assets/17284665/6586377d-f7d7-4066-8a76-242881b153ef">

We wanna to work on the OKTA stuff, which was recently added. But yeah, I will revert back for now.

**Comment by marcklingen** [comment]
Interesting. Thanks for sharing. As SSL isn’t managed by the Langfuse application container, I’m unsure how this is related to recent changes.

Do you host Langfuse without SSL/HTTPS?

How did you fix the issue (which Langfuse version works without issues for you)?

**Comment by hypernovas** [comment]
Yeah, this is a bit confusing. 

Things are more complicated to get cert and domain done here. So we started with HTTP for testing (using IP), and I am setting up the ingress to make it work properly. ==> so it would be fixed after that I assume.

We had the same setup for v2.14.6 and it worked fine.

**Comment by marcklingen** [comment]
I run Langfuse with https (Langfuse cloud) and without (locally) and could not reproduce this.

Diff of latest release, I don’t see changes that could have caused this: https://github.com/langfuse/langfuse/compare/v2.14.6...v2.15.0

in case this persist, it’d be super helpful if you could screenshare or record to help reproduce

another idea: do you run multiple application containers in parallel which are on different versions?

**Comment by ajac-zero** [comment]
I got the same error, running on an azure vm, using http with the IP. I am using the pinned version 2 container.

Running it locally on my machine works fine.

![image](https://github.com/langfuse/langfuse/assets/115488250/bd4b49d9-2f5f-4ca6-81eb-9bcc3a54c7de)

**Comment by hypernovas** [comment]
I can verify that once we set up the cert properly and use HTTPS, things are good now. Thanks @marcklingen for the help. Forcing SSL in production is not a bad idea anyway. Please feel free to close this issue.

**Comment by marcklingen** [comment]
Fixed in 2.15.1. Enforcement of HTTPS is now optional and opt-in
